SASS/style.scss

```diff
@@ -0,0 +1,629 @@
+*,
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+#offcanvasDarkNavbar{
+  background-color: #2cc9b4 !important;
+}
+body {
```

Comment:
this body class not needed please removed

Reply:
done